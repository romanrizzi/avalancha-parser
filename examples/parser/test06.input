
-- Par√©ntesis

check foo
check foo


==bar


check (foo)
check ((foo))
check (((((foo)))))

check (foo
==bar
)

check (((((foo
==bar
)))))

