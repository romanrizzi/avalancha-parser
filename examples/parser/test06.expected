[
   "program",
   [],
   [
      [
         "check",
         [
            "equal",
            [
               "var",
               "foo"
            ],
            [
               "cons",
               "True",
               []
            ]
         ]
      ],
      [
         "check",
         [
            "equal",
            [
               "var",
               "foo"
            ],
            [
               "var",
               "bar"
            ]
         ]
      ],
      [
         "check",
         [
            "equal",
            [
               "var",
               "foo"
            ],
            [
               "cons",
               "True",
               []
            ]
         ]
      ],
      [
         "check",
         [
            "equal",
            [
               "var",
               "foo"
            ],
            [
               "cons",
               "True",
               []
            ]
         ]
      ],
      [
         "check",
         [
            "equal",
            [
               "var",
               "foo"
            ],
            [
               "cons",
               "True",
               []
            ]
         ]
      ],
      [
         "check",
         [
            "equal",
            [
               "var",
               "foo"
            ],
            [
               "var",
               "bar"
            ]
         ]
      ],
      [
         "check",
         [
            "equal",
            [
               "var",
               "foo"
            ],
            [
               "var",
               "bar"
            ]
         ]
      ]
   ]
]
